oncall_slackbot/slackclient.py

```diff
@@ -27,6 +31,11 @@ def __init__(self, token, timeout=None, bot_icon=None, bot_emoji=None, connect=T
         if connect:
             self.rtm_connect()
 
+    def rtm_connect(self):
+        reply = self.webapi.rtm.connect().body
+        time.sleep(1)
```

Comment:
Why a sleep?

Reply:
copied from the method I am overriding:
https://github.com/scrapinghub/slackbot/blob/develop/slackbot/slackclient.py#L45-L48

Reply:
Actual commit, looks like it was done to "increase delays". That makes sense...

https://github.com/scrapinghub/slackbot/commit/4c8a2423e8b936a734369ce5d8ac269e0386ee67

Oh well, LGTM!

Reply:
well, I guess it does increase delays, though I am not sure why that was wanted...